### PR TITLE
Updated the CMakelists.txt to find the actual ChangeLog

### LINKED ISSUE #425: Installer fails to find ChangeLog
Hi there!
On my builds, packaging is now failing to find ChangeLog. 

Here is a little of  the log:
--------------snip--------------
==> Entering fakeroot environment...
==> Starting package()...
make: Entering directory '/z1/OS/aur/vegastrike-engine-git/src/build'
make[1]: Entering directory '/z1/OS/aur/vegastrike-engine-git/src/build'
make[2]: Entering directory '/z1/OS/aur/vegastrike-engine-git/src/build'
make[2]: Leaving directory '/z1/OS/aur/vegastrike-engine-git/src/build'
[ 30%] Built target vegastrike-engine_com
make[2]: Entering directory '/z1/OS/aur/vegastrike-engine-git/src/build'
make[2]: Leaving directory '/z1/OS/aur/vegastrike-engine-git/src/build'
[ 41%] Built target vegastrike-OPcollide
make[2]: Entering directory '/z1/OS/aur/vegastrike-engine-git/src/build'
make[2]: Leaving directory '/z1/OS/aur/vegastrike-engine-git/src/build'
[ 92%] Built target vegastrike
make[2]: Entering directory '/z1/OS/aur/vegastrike-engine-git/src/build'
make[2]: Leaving directory '/z1/OS/aur/vegastrike-engine-git/src/build'
[ 94%] Built target vegasettings
make[2]: Entering directory '/z1/OS/aur/vegastrike-engine-git/src/build'
make[2]: Leaving directory '/z1/OS/aur/vegastrike-engine-git/src/build'
[ 95%] Built target vs-buildenv-replace
make[2]: Entering directory '/z1/OS/aur/vegastrike-engine-git/src/build'
make[2]: Leaving directory '/z1/OS/aur/vegastrike-engine-git/src/build'
[ 99%] Built target vega-meshtool
make[2]: Entering directory '/z1/OS/aur/vegastrike-engine-git/src/build'
make[2]: Leaving directory '/z1/OS/aur/vegastrike-engine-git/src/build'
[ 99%] Built target vs-buildenv-asteroidgen
make[2]: Entering directory '/z1/OS/aur/vegastrike-engine-git/src/build'
make[2]: Leaving directory '/z1/OS/aur/vegastrike-engine-git/src/build'
[100%] Built target vs-buildenv-trisort
make[1]: Leaving directory '/z1/OS/aur/vegastrike-engine-git/src/build'
Install the project...
-- Install configuration: "Release"
-- Installing: /z1/OS/aur/vegastrike-engine-git/pkg/vegastrike-engine-git/usr/bin/vegastrike
-- Installing: /z1/OS/aur/vegastrike-engine-git/pkg/vegastrike-engine-git/usr/bin/vegasettings
-- Installing: /z1/OS/aur/vegastrike-engine-git/pkg/vegastrike-engine-git/usr/bin/vega-meshtool
-- Installing: /z1/OS/aur/vegastrike-engine-git/pkg/vegastrike-engine-git/usr/share/doc/Vega_Strike/AUTHORS.SF
-- Installing: /z1/OS/aur/vegastrike-engine-git/pkg/vegastrike-engine-git/usr/share/doc/Vega_Strike/AUTHORS.GITHUB
CMake Error at doc/cmake_install.cmake:54 (file):
  file INSTALL cannot find
  "/z1/OS/aur/vegastrike-engine-git/src/Vega-Strike-Engine-Source/engine/../ChangeLog":
  No such file or directory.
Call Stack (most recent call first):
  cmake_install.cmake:65 (include)

make: *** [Makefile:116: install] Error 1
make: Leaving directory '/z1/OS/aur/vegastrike-engine-git/src/build'
==> ERROR: A failure occurred in package().

I'll investigate a little further, and hopefully find a fix, but this is the current status for me.

### PR DESCRIPTION
**Pull request by evertvorster**
Thank you for submitting a pull request and becoming a contributor to the Vega Strike Core Engine.

Please answer the following:

Code Changes:
- [ ] Have the PR Validation Tests been run? See https://github.com/vegastrike/Vega-Strike-Engine-Source/wiki/Pull-Request-Validation
- [X] This is a documentation change only

Issues:
- Please list any related issues #425 

Purpose:
- What is this pull request trying to do? - Fix #425 It looks like the CMakeLists.txt was pointing at the wrong target
- What release is this for? - Any
- Is there a project or milestone we should apply this to? - ASAP. :)

### CODE REVIEW

File: doc/CMakeLists.txt [hunk 1]
```diff
@@ -9,14 +9,14 @@ IF (CMAKE_VERSION VERSION_LESS "3.14")
...
-		INSTALL(FILES "${PROJECT_SOURCE_DIR}/../ChangeLog" DESTINATION "${CMAKE_INSTALL_DOCDIR}")
+		INSTALL(FILES ChangeLog DESTINATION "${CMAKE_INSTALL_DOCDIR}")
 	ENDIF (UNIX)
 ELSE (CMAKE_VERSION VERSION_LESS "3.14")
 	# This is the preferred instruction, but it's not available until cmake 3.14
 	MESSAGE("Newer CMake version, using preferred method")
 	INSTALL(FILES AUTHORS TYPE DOC RENAME AUTHORS.SF)
 	INSTALL(FILES "${PROJECT_SOURCE_DIR}/../AUTHORS" TYPE DOC RENAME AUTHORS.GITHUB)
-	INSTALL(FILES "${PROJECT_SOURCE_DIR}/../ChangeLog" TYPE DOC)
+	INSTALL(FILES ChangeLog TYPE DOC)
```
Comment (BenjamenMeyer) [inline doc/CMakeLists.txt:19]: ahh...yeah - my bad there. Good catch.

I had a `ChangeLog` in the project root which was the git log dump; missed that it wasn't checked in.

Any how...we should reformat this file to match up, or may be just entirely re-dump it using your suggestion since the whole SVN history got migrated to git; but that can be another PR.

**Review by BenjamenMeyer — APPROVED** [review]

File: doc/CMakeLists.txt (hunk 1 above)
Comment (evertvorster) [reply doc/CMakeLists.txt:19]: Not a worry. Thanks for the speedy resolution, I'm building now to test, as well as getting some play testing in as well.
